Rubberduck.Inspections/Abstract/InspectionResultBase.cs

```diff
@@ -60,13 +60,14 @@ public string ToClipboardString()
             var documentName = Target != null 
                 ? Target.ProjectDisplayName 
                 : string.Empty;
-            if (string.IsNullOrEmpty(documentName))
-            {
-                var component = module.Component;
-                documentName = component != null 
-                    ? component.ParentProject.ProjectDisplayName 
-                    : string.Empty;
-            }
+            //todo: Find a sane way to reimplement this.
+            //if (string.IsNullOrEmpty(documentName))
+            //{
+            //    var component = module.Component;
+            //    documentName = component != null 
+            //        ? component.ParentProject.ProjectDisplayName 
+            //        : string.Empty;
+            //}
```

Comment:
Minor breaking change, don't sweat it 😉

Reply:
Actually,  nothing is calling this method.